R/quote.R

```diff
@@ -121,17 +130,87 @@ quote_identifier <-
 
 #' @rdname hidden_aliases
 #' @export
-setMethod("dbQuoteIdentifier", c("DBIConnection"), quote_identifier)
+setMethod("dbQuoteIdentifier", signature("DBIConnection"), quote_identifier)
 
 # Need to keep other method declarations around for now, because clients might
 # use getMethod(), see e.g. https://github.com/r-dbi/odbc/pull/149
 #' @rdname hidden_aliases
 #' @export
-setMethod("dbQuoteIdentifier", c("DBIConnection", "character"), quote_identifier)
+setMethod("dbQuoteIdentifier", signature("DBIConnection", "character"), quote_identifier)
 
 #' @rdname hidden_aliases
 #' @export
-setMethod("dbQuoteIdentifier", c("DBIConnection", "SQL"), quote_identifier)
+setMethod("dbQuoteIdentifier", signature("DBIConnection", "SQL"), quote_identifier)
+
+#' Unquote identifiers
+#'
+#' Call this method to convert a [SQL] object created by [dbQuoteIdentifier()]
```

Comment:
After reading this documentation I don't know what this is going to return. It sounds like it's going to return table objects which doesn't seem right?

Reply:
Return values are part of the specs in DBItest which I haven't adapted yet.

The method is returning a `Table` object. Why does this seem wrong?

Reply:
Root of my confusion here is that `Table` is a reference/identifier, not a table.